tests/console/clamav.pm

```diff
@@ -40,6 +40,17 @@ sub run {
     $self->select_serial_terminal;
 
     zypper_call('in clamav vim');
+    zypper_call('info clamav');
+
+    # Check Clamav version
+    # Jira ID SLE-16780: upgrade Clamav SLE
+    my $current_ver = script_output("zypper info clamav | grep Version | cut -d':' -f2| cut -d'-' -f1");
+    record_info("Clamav_ver", "Current Clamav package version: $current_ver");
+
+    if ($current_ver < 0.101) {
```

Comment:
Should we add product check, e.g.,
if (is_sle('>=15-sp3')) {
    if ($current_ver < 0.101) {

Reply:
Good point, it will avoid that other product run this job as well. This is currently specific for SP3. Thanks @lilyeyes